kore/src/Kore/Internal/SideCondition.hs

```diff
@@ -281,6 +281,23 @@ addAssumptions predicates sideCondition =
             predicates <> assumedTrue sideCondition
         }
 
+areIncludedIn ::
+    Eq variable =>
+    Foldable f =>
+    f (Predicate variable) ->
+    SideCondition variable ->
+    Bool
+areIncludedIn predicates sideCondition =
+    all (flip isIncludedIn sideCondition) predicates
```

Comment:
If `isIncludedIn` is only used here, does it make sense to flip the arguments here instead of simply flipping the arguments in the definition?  Of course the name would need to be modified.

Reply:
I wanted it to be similar to the `elem` function, so that the order of the arguments is intuitive.